mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomeConfig.java

```diff
@@ -1601,7 +1606,8 @@ public void remove() {
         public void setOnReloadListener(OnReloadListener listener);
     }
 
-    // UUIDs used to create PanelConfigs for default built-in panels. These are
+    //
```

Comment:
What's happening here?

Reply:
adding UUIDS, generated random, I don't know how, they stored it fixed in the code.
but there is some others parts in the code may related to this, like "UUIDUtil.java" but they don't use it.
so I generate a key for MYOFFRZ tab like it.
 with 8-4-4-4-12 digits.